gradle.properties

```diff
@@ -26,4 +26,4 @@ systemProp.gradle.internal.testdistribution.queryResponseTimeout=PT20S
 systemProp.org.gradle.kotlin.dsl.precompiled.accessors.strict=true
 
 # Default performance baseline
-defaultPerformanceBaselines=7.6-commit-c6141aed
+defaultPerformanceBaselines=7.6-commit-58171cb5d87
```

Comment:
This is the second change that is not identical to the backported PRs. It uses the commit ID of the commit right before the one making the change, similar to what was done in the original PR.